music21/musicxml/xmlToM21.py

```diff
@@ -1832,7 +1826,7 @@ def updateTransposition(self, newTransposition):
 
         if self.activeInstrument is not None:
             if (self.activeInstrument.transposition is None
-                    and self.firstMeasureParsed is False):
+                and self.firstMeasureParsed is False):
```

Comment:
the second line of a parenthesized "if" clause is generally overindented to avoid looking like part of the body.